### PR TITLE
Add Luciano Ramalho keynote speaker blog

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
Solves https://github.com/pythonindia/inpycon2021-tasks/issues/58

File: content/blog/2021/announcing-keynote-speaker-luciano.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Title: Announcing our fourth Keynote Speaker for PyCon India 2021 - Luciano Ramalho
+Date: 2021-07-01 17:00
+Category: 2021
+Slug: keynote-announcement-luciano-ramalho
+Summary: We are happy to announce **Luciano Ramalho** as a Keynote Speaker for PyCon India 2021!
+
+<p align="center"><img alt="Luciano Ramalho" src="https://in.pycon.org/2021/images/luciano.png"></p>
+
+Luciano Ramalho is the author of Fluent Python (O'Reilly), published in 9 languages. He's been a professional Python programmer since 1998, and he just finished writing the Fluent Python, Second Edition, covering Python 3.9 and 3.10—to be released in Q4, 2021. Ramalho is a Principal Consultant at ThoughtWorks, and co-founder of Garoa.net.br, a hackerspace in São Paulo, Brazil.
```
Comment (raukadah) [inline content/blog/2021/announcing-keynote-speaker-luciano.md:9]: Can we hyperlink the fluent python book link?

**Review by raukadah — COMMENTED** [review]
just one comment, looks nit

File: content/blog/2021/announcing-keynote-speaker-luciano.md (hunk 1 above)
Comment (raukadah) [reply content/blog/2021/announcing-keynote-speaker-luciano.md:9]: Let's add it later on. approving currently.

**Review by raukadah — APPROVED** [review]

**Pull request merged**